MoneyDataType/BitCoin.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OrchardCore.Commerce.Abstractions;
+
+namespace OrchardCore.Commerce.Money
+{
+    public struct BitCoin : ICurrency
+    {
+        public string Symbol => "₿";
+        public string EnglishName => "BitCoin";
+        public string NativeName => "BitCoin";
+        public string IsoCode => "BTC";
+        public int DecimalPlaces => 8;
+
+        public string Culture => "BTC";
+
+        public bool Equals(ICurrency other) => other.IsoCode == IsoCode;
+
+        public string ToString(decimal amount) => amount.ToString($"N{DecimalPlaces}") + " BTC";
```

Comment:
why not use the configured decimal places?  I'm against hard-coded values in a core module like this.

Reply:
Because it's a constant anyways. String interpolation is very inefficient for that sort of thing. If you really want to make it dynamic (but it seems way overkill to me), please do `'N'+DecimalPlaces` instead.